goa/Classes/mini_games/Chain.cpp

```diff
@@ -41,7 +41,7 @@ GraphemeGrid* Chain::createGraphemeGrid(GLfloat width, GLfloat height, int numRo
 
 Node* Chain::loadNode() {
 	std::string chainSceneType[] = { "monkey","elephant","flamingo" };
-	Chain::_SS = chainSceneType[RandomHelper::random_int(0, 2)];
+	Chain::_SS = chainSceneType[RandomHelper::random_int(1, 2)];
```

Comment:
Just add a comment here to reason why 0 was changed to 1

Reply:
Okay